js-core-1/week-2/lesson.md

```diff
@@ -444,20 +457,14 @@ In the node REPL, enter the following array:
 > let fruits = ['banana', 'apple', 'strawberry', 'kiwi', 'fig', 'orange'];
 ```
 
-1. Which index do you need to return:
+Which index do you need to return:
```

Comment:
This question is a bit ambiguous. I read it as though I'm being asked to return an index. 

How about:

> Now, using the correct indexes, get the following values from the array: